projects/storefrontlib/src/lib/checkout/components/multi-step-checkout/review-submit/review-submit.component.spec.ts

```diff
@@ -12,6 +12,7 @@ import * as fromAuth from '../../../../auth/store';
 
 import { CheckoutService } from '../../../services/checkout.service';
 import { CartDataService } from '../../../../cart/services/cart-data.service';
+import { UserService } from '../../../../user/facade/user.service';
```

Comment:
Can you remove `StoreModule.forFeature('user', fromUser.getReducers()),` from line 101?

Reply:
And also maybe switch to using the auth facade?